docs/appservice.md

```diff
@@ -2,31 +2,65 @@ Draupnir can be run as an appservice, allowing users you trust or on your homese
 
-4. Generate the appservice registration file. This will be used by both the appservice and your homeserver.
+5. Generate the appservice registration file. This will be used by both the appservice and your homeserver.
    Here, you must specify the direct link the Matrix Homeserver can use to access the appservice, including the Matrix port it will send messages through (if this bridge runs on the same machine you can use `localhost` as the `$HOST` name):
 
    `docker run --rm -v /your/path/to/draupnir-data:/data gnuxie/draupnir appservice -r -u "http://$HOST:$MATRIX_PORT" -f /data/config/draupnir-registration.yaml`
 
-5. Step 4 created an application service bot. This will be a bot iwth the mxid specified in `draupnir-registration.yaml` under `sender_localpart`. You now need to invite it in the access control room that you have created in Step 1.
+6. Create a `config/config.production.yaml` file that can be copied from the example in `config/default.yaml` just like in step 4. The file also needs to be accessible to the container so whatever path is used for `config/config.appservice.yaml` can be reused to also house this file.
 
-6. Start the application service `docker run -v /your/path/to/draupnir-data/:/data/ gnuxie/draupnir appservice -c /data/config/config.appservice.yaml -f /data/config/draupnir-registration.yaml -p $MATRIX_PORT`
+The provisioned Draupnirs only inherit a subset of the configuration options that are accessible to Bot mode Draupnir. Those are the following options. If there's `:` as a suffix to a config option that means there are sub options like how under commands in the default config you also find `additionalPrefixes:` with a value of `draupnir`.
+```
+logLevel
+syncOnStartup
+fasterMembershipChecks
+automaticallyRedactForReasons:
+protectAllJoinedRooms
+backgroundDelayMS
+commands:
+protections:
+```
 
-7. Copy the `draupnir-registration.yaml` to your matrix homeserver and refer to it in `homeserver.yaml` like so:
+7. Start the application service `docker run -v /your/path/to/draupnir-data/:/data/ gnuxie/draupnir appservice -c /data/config/config.appservice.yaml -f /data/config/draupnir-registration.yaml -p $MATRIX_PORT --draupnir-config /data/config/production.yaml`
+
+8. Copy the `draupnir-registration.yaml` to your matrix homeserver and refer to it in `homeserver.yaml` like so:
 ```
   app_service_config_files:
     - "/data/draupnir-registration.yaml"
 ```
+
+# Post Install Setup and Configuration.
+
+If you successfully followed Steps 1-8 or got to the point your running the bot thru another mean congratulations. Time for Post Install Finalisation and Configuration of your new appservice.
```

Comment:
`thru`

Reply:
>Time for Post Install Finalisation and Configuration of your new appservice.

Then there are some post install steps and configuration that you should apply to your new appservice